libs/sdk-ui-kit/src/AppearancePage/WhiteLabeling.tsx

```diff
@@ -0,0 +1,73 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { Button } from "../Button";
+import { TooltipIcon } from "./TooltipIcon";
+import { DocumentationLink } from "./DocumentationLink";
+import { LoadingSpinner } from "../LoadingSpinner";
+import { StylingComponent } from "./StylingComponent";
+import { StylingHeader } from "./StylingHeader";
+import { StylingFooter } from "./StylingFooter";
+
+const ALIGN_POINTS = [{ align: "cr tl" }, { align: "bc tc" }, { align: "bc tr" }];
+const ARROW_OFFSETS = { "cr tl": [16, -12], "bc tc": [0, 16], "bc tr": [12, 16] };
+
+/**
+ * @internal
+ */
+export interface IWhiteLabelingProps {
+    title: string;
+    status: string;
+    tooltip: string;
+    linkText: string;
+    linkValue: string;
+    actionText: string;
+    isLoading: boolean;
+    onSubmit: () => void;
+}
+
+/**
+ * @internal
+ */
+export const WhiteLabeling = (props: IWhiteLabelingProps) => {
+    const { title, status, tooltip, linkText, linkValue, actionText, isLoading, onSubmit } = props;
+
+    return (
+        <StylingComponent>
+            <StylingHeader>
+                <span className="gd-appearance-styling-title">{title}</span>
+                <TooltipIcon
+                    text={tooltip}
+                    iconClass="gd-icon-circle-question"
+                    alignPoints={ALIGN_POINTS}
+                    arrowOffsets={ARROW_OFFSETS}
+                />
+                {isLoading ? (
+                    <LoadingSpinner className="small gd-loading-equalizer-spinner" />
+                ) : (
+                    <span className="gd-white-labeling-status">{status}</span>
+                )}
+            </StylingHeader>
+            <div className="gd-appearance-styling-separator" />
```

Comment:
Based on the logic with how everything is wrapped as a component and reused between widgets, this should be its own component as well (and used in the both widgets).

Possibly it already exists in form of `ContentDivider`? Maybe this component can be reused somehow.